### PR TITLE
Counting Leaderboard

### PR DESCRIPTION
**Pull request by tanndlin**
Only increments existing documents in the collection

I ran a one-time script to count up the messages and populate that collection

### CODE REVIEW

File: src/common/types.ts [hunk 1]
```diff
@@ -229,3 +229,8 @@ export interface IUserPoints {
   numPoints: number;
   lastKingCrowning: Date;
 }
+
+export interface ICountingEntry {
```
Comment (c650) [inline src/common/types.ts:233]: no guildID ?

File: src/app/handlers/counting.handler.ts [hunk 2]
```diff
@@ -38,6 +39,9 @@ export class CountingHandler extends Handler {
         await message.react(this._buzzEmoji);
       }
 
+      // Increment count in counting leaderboard
+      const userDoc = await this._getUserCountingDoc(message.author.id);
+      await CountingLeaderboardModel.updateOne(userDoc, { $inc: { count: 1 } });
```
Comment (c650) [inline src/app/handlers/counting.handler.ts:44]: is there no updateOrInsert type thing? ?

**Review by msabur — CHANGES_REQUESTED** [review]
There’s a typo in filename `coutingLeaderboard.plugin.ts`.

File: src/app/handlers/counting.handler.ts (hunk 2 above)
Comment (tanndlin) [reply src/app/handlers/counting.handler.ts:44]: I tried, but I don't think I can do it with an insert too

**Pull request merged**